app/components/no_results.js

```diff
@@ -66,15 +68,24 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
             paddingHorizontal: 15,
         },
         title: {
-            color: changeOpacity(theme.centerChannelColor, 0.4),
+            color: changeOpacity(theme.centerChannelColor, 0.8),
             fontSize: 20,
-            fontWeight: '600',
+            fontWeight: 'bold',
```

Comment:
This may have to be reverted back to `'600'`, as the font in this section is supposed to be semibold (reference in [this comment](https://github.com/mattermost/mattermost-mobile/pull/4641#pullrequestreview-460946137) on a very similar PR I worked on recently)

Reply:
Done